cdkworkshop.com/cdkworkshop.com.ts

```diff
@@ -102,24 +103,29 @@ export class CdkWorkshop extends Stack {
 
         // CloudFront distribution
         const cert = acm.Certificate.fromCertificateArn(this, 'Certificate', props.certificate);
-        const cdn = new cloudfront.CloudFrontWebDistribution(this, 'CloudFront', {
-            viewerProtocolPolicy: cloudfront.ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
+        const cdn = new cloudfront.Distribution(this, 'CloudFrontDistribution', {
+            defaultBehavior: {
+                viewerProtocolPolicy: cloudfront.ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
+                origin: new cloudfrontOrigins.S3Origin(bucket, {
+                    originAccessIdentity: origin, originPath: `/${contentHash}`
+                }),
+                cachePolicy: new cloudfront.CachePolicy(this, 'CachePolicy', {
+                    maxTtl,
+                }),
+                responseHeadersPolicy: new cloudfront.ResponseHeadersPolicy(this, 'ResponseHeadersPolicy', {
```

Comment:
Why not use `cloudfront.ResponseHeadersPolicy.SECURITY_HEADERS`?

Reply:
This policy is more strict than `cloudfront.ResponseHeadersPolicy.SECURITY_HEADERS`.
* We're setting `X-Frame-Options` to `DENY` instead of `SAMEORIGIN`.
* The managed policy doesn't override the origin except for `X-Content-Type-Options`; we're doing it for all of the headers
* We're setting `Strict-Transport-Security` with `includeSubDomains`, the managed policy does not